packages/server/src/services/agent.service.ts

```
@@ -38,35 +39,47 @@ export class AgentService implements IAgentService {
       message: 'Processing agent request',
       request: userRequest.user_request,
     });
-
     try {
-      let result: Promise<unknown>;
+      let result: any;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)